content/observability-slurm/README.md

```diff
@@ -0,0 +1,693 @@
+# Installing observability stack on SLURM
+
+## Automatic
+The easiest way of creating SLURM cluster with observability stack enabled is to use our Terraform/Ansible script. Make sure to enable observability in your values:
+```hcl
+enable_observability = true
+```
+
+## Manual steps
+
+### Prerequisites
+- SLURM Cluster: A functioning SLURM cluster with head nodes and compute nodes.
+- Administrative Access: Root or sudo privileges on all nodes.
+- Internet Access: Nodes must be able to download packages from the internet.
+- NVIDIA GPUs: Compute nodes with NVIDIA GPUs and drivers properly installed.
+- Operating System: Ubuntu 20.04 or newer
+- Firewall: Ability to configure firewall rules for required ports
+
+### Security Considerations
+Before proceeding with installation, please note:
+- Configure firewall rules to restrict access to the following ports:
+  - Prometheus (9090)
+  - Node Exporter (9100)
+  - DCGM Exporter (9400)
+  - Grafana (3000)
+- Use SSL/TLS certificates for secure communication
+- Use secrets management for sensitive credentials
+- Implement authentication for all services
+- Regular security updates and monitoring
+
+### Installation steps
+
+#### 1. *Update System and Install Dependencies*
+   
+_On all nodes_
+
+Update package lists and install required packages:
+```bash
+$ sudo apt update
+$ sudo apt install -y curl wget gnupg2 software-properties-common apt-transport-https ca-certificates
+```
+
+#### 2. *Create Users and Directories*
+
+_On all nodes_
```

Comment:
For commands that are required to be run on all compute nodes, can we recommend using a tool like `pssh` or clush ( https://clustershell.readthedocs.io/en/latest/tools/clush.html ) to make it easier to get started with large cluster management? Perhaps this warrants another mini section on setting up a hostfile for parallel VM management